presto-main/src/main/resources/webapp/index.html

```diff
@@ -172,7 +176,9 @@
                             <div className="col-md-12"><a href={"query.html?" + query.queryId}>{query.queryId}</a></div>
                         </div>
                         <div className="row">
-                            <div className="col-md-12 truncate">{query.session.user}</div>
+                            <div className="col-md-12 truncate">
+                            <span>{query.session.user}</span> <span>{authenticatedIcon}</span>
```

Comment:
Do the `span` tags here do anything? They are normally only needed to create a DOM element for attaching classes or other info, but that doesn't seem to be the case here.